src/main/java/ch/uzh/ifi/hase/soprafs23/service/GameService.java

```diff
@@ -61,4 +61,23 @@ public void nextTurn(int accessCode, int scoredPoints) {
         gameRepository.save(existingGame);
         gameRepository.flush();
     }
+    public PlayerRole getPlayerRole(int accessCode,String userName){
+        Game existingGame = gameRepository.findByAccessCode(accessCode);
```

Comment:
I'd raise an error if we can't find a game with the given accessCode. I did this all the time in my code. You can copy the code from there.